plugins/hypervisors/vmware/src/main/java/com/cloud/hypervisor/vmware/VmwareServerDiscoverer.java

```diff
@@ -367,10 +367,6 @@ public VmwareServerDiscoverer() {
                 details.put("url", hostMo.getHostName());
                 details.put("username", username);
                 details.put("password", password);
-                boolean uefiLegacySupported = hostMo.isUefiLegacySupported();
```

Comment:
I think these could be kept, same for other discoverers

Reply:
It would be redundant as the code to check the UEFI support is subsequently called while adding a host